src/auto/accel_group.rs

```diff
@@ -35,9 +36,12 @@ impl AccelGroup {
         }
     }
 
-    //pub fn from_accel_closure(closure: /*Ignored*/&glib::Closure) -> Option<AccelGroup> {
-    //    unsafe { TODO: call ffi::gtk_accel_group_from_accel_closure() }
-    //}
+    pub fn from_accel_closure(closure: &glib::Closure) -> Option<AccelGroup> {
```

Comment:
Instead of having a plain `GClosure` here it would be nicer to implement this manually around the actual type that is expected from the closure. `Fn(&AccelGroup, &Object, u32, gdk::ModifierType) -> bool` (`GtkAccelGroupActivate`) from what I can see

Reply:
Since this one is supposed to return an `AccelGroup` for the matching `Closure`, I wonder it would work then...

Reply:
It wouldn't indeed, unless we let `gtk_accel_group_connect()` return the closure that we internally created. This API is really nice :man_facepalming: